src/Pagination/Doctrine/Dbal/PagerIterator.php

```diff
@@ -0,0 +1,108 @@
+<?php declare(strict_types=1);
+
+namespace Fazland\ApiPlatformBundle\Pagination\Doctrine\Dbal;
```

Comment:
`DBAL` all uppercase instead of `Dbal` (like the Doctrine's one).

Edit: I just notice that you also used DBAL in the previous PR in DoctrineExtra :)
For instane:
https://github.com/fazland/doctrine-extra/blob/master/src/DBAL/IteratorTrait.php#L3

Reply:
👍